.gitignore

```diff
@@ -31,6 +31,7 @@ include/generated/*
 iroha.conf
 peers.list
 cmake-build*
+*vcpkg-build*
```

Comment:
It was made because of documentation change.
```
git clone https://github.com/hyperledger/iroha.git
cd iroha
./vcpkg/build_iroha_deps.sh $PWD/vcpkg-build
```